<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>711</width>
    <height>421</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton#dm0 { 
	border:6px solid black;
}

QPushButton#dm1{
	border:6px solid black;
}
QPushButton#dm2{
	border:6px solid black;
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="srtLayout">
      <item>
       <widget class="QToolBox" name="toolBox">
        <property name="contextMenuPolicy">
         <enum>Qt::NoContextMenu</enum>
        </property>
        <property name="acceptDrops">
         <bool>false</bool>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <property name="tabSpacing">
         <number>6</number>
        </property>
        <widget class="QWidget" name="srt0">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>140</width>
           <height>126</height>
          </rect>
         </property>
         <attribute name="label">
          <string>인적 사항</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QVBoxLayout" name="srtBox0"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="srt1">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>533</width>
           <height>412</height>
          </rect>
         </property>
         <attribute name="label">
          <string>학력 사항</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QVBoxLayout" name="srtBox1"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="srt2">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>533</width>
           <height>412</height>
          </rect>
         </property>
         <attribute name="label">
          <string>경력 사항</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QVBoxLayout" name="srtBox2"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="srt3">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>533</width>
           <height>412</height>
          </rect>
         </property>
         <attribute name="label">
          <string>활동 및 수상 경력</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <layout class="QVBoxLayout" name="srtBox3"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="srt4">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>533</width>
           <height>412</height>
          </rect>
         </property>
         <attribute name="label">
          <string>자격증</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <layout class="QVBoxLayout" name="srtBox4"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="srt5">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>533</width>
           <height>412</height>
          </rect>
         </property>
         <attribute name="label">
          <string>프로젝트</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <layout class="QVBoxLayout" name="srtBox5"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="srt6">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>533</width>
           <height>412</height>
          </rect>
         </property>
         <attribute name="label">
          <string>자기소개서</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <layout class="QVBoxLayout" name="srtBox6"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="srt7">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>533</width>
           <height>412</height>
          </rect>
         </property>
         <attribute name="label">
          <string>포트폴리오</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <item>
           <layout class="QVBoxLayout" name="srtBox7"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="srt8">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>533</width>
           <height>412</height>
          </rect>
         </property>
         <attribute name="label">
          <string>기타</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_9">
          <item>
           <layout class="QVBoxLayout" name="srtBox8"/>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QTabWidget" name="docTab">
        <property name="minimumSize">
         <size>
          <width>543</width>
          <height>360</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>543</width>
          <height>360</height>
         </size>
        </property>
        <property name="currentIndex">
         <number>-1</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
   <widget class="QLabel" name="intro">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>40</y>
      <width>521</width>
      <height>321</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>521</width>
      <height>321</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>521</width>
      <height>321</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>intro</zorder>
   <zorder></zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>711</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
